### ISSUE TITLE
Dark theme elements are broken

### ISSUE THREAD

**Issue by vladimirkosolapov** [veler/DevToys#46]
**Describe the bug**
The last app update broke the dark theme elements. Some of these bugs are shown in the screenshots (plus some notes and a bug in «TextDiff»):

**Screenshots**
<details>
  <summary>Click to expand</summary>
  <img src="https://user-images.githubusercontent.com/34649010/138918808-13d00154-463c-4ee5-a582-871e455985dc.png"/>
  <img src="https://user-images.githubusercontent.com/34649010/138918919-9f40b9e7-2a00-466b-b7dd-80998bc032e8.png"/>
  <img src="https://user-images.githubusercontent.com/34649010/138919008-6d048f6f-0a47-4065-a4b9-70c993a82200.png"/>
</details>

**Configuration:**
 - OS & Version: Windows 11 (22000.282)
 - Theme: Dark
 - DevToys app version 0.6.5.0

**Comment by veler** [comment]
@vladimirkosolapov , I think I have a proper fix for the dark theme now. I will commit it soon.

For the issue where the ComboBox appears below the title of the setting: it is by design, to avoid ending up with a UI like this:

![image](https://user-images.githubusercontent.com/3747805/139505885-245bc536-d49d-4395-97a2-1cac325ac317.png)

Instead, we prefer having the following:
![image](https://user-images.githubusercontent.com/3747805/139506212-032a248b-bbe5-466b-a595-6f9e7670566a.png)

In the update we shipped at the beginning of the week, we updated the layout by increasing these margins here in order to be more compliant Windows 11 guideline:
![image](https://user-images.githubusercontent.com/3747805/139506027-57f1600a-1235-4106-95e4-b3e3c58661bb.png)

By design, when the UI get smaller, the ComboBox will go below the Title and Description of the setting when the **[Size of ComboBox] > [1/3 the Setting size]**.

In the screenshot you shared, the ComboBox is indeed larger than 1/3 of the total width of the setting control.

![image](https://user-images.githubusercontent.com/3747805/139506429-d9009bfd-8d2e-42ea-8fd6-c6b452e7193f.png)

In conclusion, the reason why you see a difference if because we changed the margins around the setting control, and now the setting control enters in a compact state depending on the length of the text in the ComboBox. It is the expected behavior. :)
